inc/agl.inc

```diff
@@ -4,53 +4,31 @@ LICENSE = "MIT"
 
 inherit build_yocto
 inherit xt_quirks
-require xt_rcar.inc
 
 S = "${WORKDIR}/repo"
-# path for machine specific layers and recipes
-FILESEXTRAPATHS_prepend := "${THISDIR}/../machine:"
+
+################################################################################
+# set to AUTOREV so we can override this while reconstructing this build with
+# specific commit ID
+################################################################################
+SRCREV = "${AUTOREV}"
```

Comment:
I can't see description of this change in the commit message.

Reply:
this is just a code move inside the same file
will put it where it was